main.tf

```diff
@@ -1,65 +1,65 @@
-provider "aws" {
+  provider "aws" {
   region = "us-east-1"
 
 }
 
-resource "aws_instance" "app-server" {
-  instance_type = var.ec2-instance
+resource "aws_instance" "app-server" {          //zz problem, fix this by enabling kk for a resource with a module path: root             || for Non-attribute problems
```

Comment:
Minor thing: but "resource with a module path: root " looks very unprofessional when it is root. Hopefully we cna exclude that part for root.

Reply:
(root) 
it would be like this, as root has no name